docs/api-gateway.md

```diff
@@ -140,9 +140,9 @@ Configuration of such API endpoint might look like this:
 }]
 ```
 
-> Important field is `target`. `target` describes what event stream to use. Currently it's either Kafka or Kinesis. HTTP method and `path` name are up to your choice.
+Note that the `target` field is set to `kafka` (for Kinesis use `kinesis`).
 
-This type of endpoint requires specific HTTP request body:
+The endpoint expects the following request format:
 
 ```json
 {
```

Comment:
Perhaps here, we should mention current format/approach. This one is deprecated if I understood correctly.

Reply:
good catch, will update that